core/dbt/task/seed.py

```diff
@@ -62,10 +62,6 @@ def print_result_line(self, result):
 
 
 class SeedTask(RunTask):
-    def defer_to_manifest(self, adapter, selected_uids):
```

Comment:
Discussed offline with @MichelleArk: Should we actually still disable deferral for tasks like seed/freshness/list, since those tasks never (in theory) resolve runtime references?

> It's technically possible to have a seed reference another node, via a pre/post hook. Or write custom code in a project (on-run-*) hook that wants to use the `defer_relation` property of nodes in the graph. I don't think that's _good_, but it is possible.
>
> At this point it feels like a choice between consistency, or a (small) performance optimization. Either way, I want all commands to support the flags as CLI options. We could keep the overrides in for now, until the follow-up refactor (https://github.com/dbt-labs/dbt-core/pull/9199) that removes deferral logic from tasks entirely, and makes it part of manifest loading. That does feel like the right direction, and this is a step on the way there.